ide/coqide/preferences.ml

```diff
@@ -666,10 +666,47 @@ let tag_button () =
 let pref_file = Filename.concat (Minilib.coqide_config_home ()) "coqiderc"
 let accel_file = Filename.concat (Minilib.coqide_config_home ()) "coqide.keys"
 
+let accel_regex = Str.regexp "\\(; \\|\\)(gtk_accel_path \"\\([^\"]*\\)\""
```

Comment:
You can use string literals like `{| my \ string |}` in OCaml. This let's you avoid having to escape so much.

Reply:
Yeah, I tried that.  Unfortunately, it confuses my IDE in the rest of the file after {| ... |} because it has an odd number of quotation marks.

![image](https://user-images.githubusercontent.com/1253341/171553868-bfef4246-c366-424e-99b4-0f1b7c4a61a7.png)

But tweaking it to this works without changing the semantics, so I'll do that:

![image](https://user-images.githubusercontent.com/1253341/171554698-cdb5677f-1952-4411-9ed7-3d6d6e7dcec3.png)